zerver/tests/test_auth_backends.py

```diff
@@ -1983,7 +1983,11 @@ def social_auth_test(
         return result
 
     def generate_saml_response(
-        self, email: str, name: str, extra_attributes: Mapping[str, List[str]] = {}
+        self,
+        email: str,
+        name: str,
+        extra_attributes: Mapping[str, List[str]] = {},
+        include_session_index: bool = True,
```

Comment:
This commit needs to fix up `docs/production/authentication-methods.md` where it talks about the lack of `SessionIndex` support, in the caveats.

Reply:
Ah nice catch, fixed